cellfinder_napari/tests/test_cellfinder_napari.py

```diff
@@ -7,51 +7,33 @@
 from PIL import Image
 
 from cellfinder_napari.detect import detect
-
-test_data_dir = pathlib.Path(__file__) / ".." / ".." / ".." / "test_data"
-
-
-def load_stack(directory: pathlib.Path) -> np.ndarray:
-    """
-    Load a stack of .tif image files in the given directory.
-    """
-    tif_glob = str((directory / "*.tif").resolve())
-    ims = []
-    for f in sorted(glob.glob(tif_glob)):
-        ims.append(np.array(Image.open(f)))
-    return np.stack(ims, axis=0)
-
-
-# @pytest.fixture
-# def signal_image():
-#    return load_stack(test_data_dir / "crop_planes" / "ch0")
-
-
-# @pytest.fixture
-# def background_image():
-#    return load_stack(test_data_dir / "crop_planes" / "ch1")
-
-
-def test_detect_function(
-    make_napari_viewer,
-):  # , signal_image, background_image):
-    # Smoke test that running cell detection from the widget works
-
-    # Create viewer and add images
```

Comment:
Is this part of the PR? I've seen this before, and potentially something to add to our developer conventions. To me, it looks like this is tidying up the tests and IMO should be part of a different PR

Reply:
Yes, tidying up the tests is part of this PR. A lot of the code I deleted is redundant because it's easy to get data via. the new `sample_data` function.

Reply:
Okay, I guess the title "Add ..." is a bit misleading